// *****************************************************************************
//
// This file is part of a MASA library or program.
// Refer to the included end-user license agreement for restrictions.
//
// Copyright (c) 2010 MASA Group
//
// *****************************************************************************

package MsgsClientToReplay;

import "Common.proto"; 

message MsgControlSkipToTick {
   required int32 tick = 1;
}

message MsgClientToReplay {
    message Content {
       optional MsgControlSkipToTick              control_skip_to_tick       = 1;
       optional Common.MsgControlChangeTimeFactor control_change_time_factor = 2;
       optional Common.MsgControlStop             control_stop               = 3;
       optional Common.MsgControlPause            control_pause              = 4;
       optional Common.MsgControlResume           control_resume             = 5;
    }
    optional int32   context = 1;
    required Content message = 2;
}
